client/layout/sidebar/custom-icon.jsx

```diff
@@ -13,20 +13,21 @@
  */
 import React from 'react';
 
-const SidebarCustomIcon = ( { alt, className, icon, ...rest } ) => {
+const SidebarCustomIcon = ( { icon, ...rest } ) => {
 	if ( ! icon ) {
 		return null;
 	}
 
 	if ( icon.indexOf( 'data:image' ) === 0 || icon.indexOf( 'http' ) === 0 ) {
```

Comment:
Should we separate these conditions and only change icons that include `data:image` (svgs) to `spans`? It seems like we allow non-svg icons from third-party plugins, like NextGEN Gallery:
Before | After
------ | ----
<img width="273" alt="Screen Shot 2021-04-05 at 3 39 29 PM" src="https://user-images.githubusercontent.com/1689238/113618087-2abe4100-9625-11eb-902c-9bfdd97f6864.png"> | <img width="273" alt="Screen Shot 2021-04-05 at 3 29 23 PM" src="https://user-images.githubusercontent.com/1689238/113617803-ce5b2180-9624-11eb-995d-f79a65f9cd87.png">

Reply:
Yes, I'm not sure what my logic was there. The equivalent in Jetpack is targeted to icons defined as `data:image/svg+xml` so I think we should do the same here. I'll update it.